rock_paper_scissors/rps.py

```diff
@@ -3,6 +3,31 @@
 import sys
 
 def rock_paper_scissors(n):
+  # PR(n,r) = n^r
+  # if r == 0:
+  #   return 0
+  # if r == 1:
+  #   return 3
+  # if r == 2:
+  #   return 9
+  # if r == 3:
+  #   return 27
+  if n == 1:
+    return [['rock'], ['paper'], ['scissors']]
+  rock_list = ['rock']
+  paper_list = ['paper']
+  scissors_list = ['scissors']
+  full_list = [['rock'], ['paper'], ['scissors']]
+  listed = []
+    # concat = full_list*3
+  list_multiplier = 3**n//3
+  print(list_multiplier)
+  big_list = full_list*list_multiplier
+  def concat_rock(the_list, x):
+      
+    the_list[x].append('rock')
+    print(the_list)
+  concat_rock(big_list, 0)
```

Comment:
So the tricky thing here is you need to create an empty list and a list of each of rock, paper, and scissors, then you'll want an inner recursive loop that counts down the rounds left and calls itself, with one fewer round left (inside a for loop that iterates over the RPS list). Call the inner function with `n` and an empty result list. Then return the output (first) list. Sorry for the run-on sentence.